sdk/keyvault/keyvault-certificates/samples/javascript/contacts.js

```diff
@@ -15,8 +15,7 @@ async function main() {
   // - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
   // - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
   // - AZURE_CLIENT_SECRET: The client secret for the registered application
-  const vaultName = process.env["KEYVAULT_NAME"] || "<keyvault-name>";
-  const url = `https://${vaultName}.vault.azure.net`;
+  const url = process.env["KEYVAULT_URI"] || "<keyvault-url>";
```

Comment:
Changed this to the `KEYVAULT_URI` that comes back from the ARM template. This enables samples to run in sovereign clouds without modification.

Reply:
There should be documentation associated with this environment variable, for example in sample.env, we instruct users to set the `KEYVAULT_NAME` env var. If we switch to `KEYVAULT_URI`, we need to make sure to update those instructions and that it's clear what value to use and from where a user gets it.